### DISCUSSION TITLE
Website: RSS feed is broken.

### DISCUSSION THREAD

**Discussion by midijohnny** [msteveb/jimtcl#317] [General]
The [website] for jimtcl features an [RSS feed] - this is currently broken. (2024-OCT-09).
It appears to be delivering server-side code directly, rather than rendering to XML.

```
curl https://jim.tcl.tk/index.html/doc/www/www/feed.xml
<%= atom_feed :limit => @config[:rss_feed_count].to_i %>
```

[website]: https://jim.tcl.tk/index.html/doc/www/www/index.html
[RSS feed]: https://jim.tcl.tk/index.html/doc/www/www/feed.xml

**Comment by msteveb** [comment]
Thanks. I’ll take a look

**Reply by msteveb** [reply]
Now fixed
